ci_reporter_cucumber.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features|acceptance)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "cucumber", "~> 1.3.3"
+  spec.add_dependency "cucumber", "~> 2.4.0"
```

Comment:
There's no need to be as drastic with the change here. It appears that `">= 1.3.3", "< 3.0.0"` should be fine.